core/include/drivers/pl022_spi.h

```diff
@@ -29,27 +29,40 @@
 #ifndef __PL022_SPI_H__
 #define __PL022_SPI_H__
 
+#include <gpio.h>
 #include <spi.h>
-#include <types_ext.h>
 
 #define PL022_REG_SIZE	0x1000
 
+enum pl022_cs_control {
+	PL022_CS_CTRL_BUILTIN,
```

Comment:
minor: `PL022_CS_CTRL_GPIO` is more explicit than `PL022_CS_CTRL_BUILTIN` as it is GPIO based.